generator/src/lib.rs

```diff
@@ -2464,15 +2468,137 @@ pub fn write_source_code(path: &Path) {
+    std::fs::create_dir_all(&dir).expect("failed to create vk dir");
+
+    let mut vk_rs_file = File::create(vk_rs).expect("vk.rs");
+
+    let mut vk_prelude_file = File::create(dir.join("prelude.rs")).expect("vk/prelude.rs");
+    let mut vk_features_file = File::create(dir.join("features.rs")).expect("vk/features.rs");
+    let mut vk_definitions_file =
+        File::create(dir.join("definitions.rs")).expect("vk/definitions.rs");
+    let mut vk_enums_file = File::create(dir.join("enums.rs")).expect("vk/enums.rs");
+    let mut vk_bitflags_file = File::create(dir.join("bitflags.rs")).expect("vk/bitflags.rs");
+    let mut vk_constants_file = File::create(dir.join("constants.rs")).expect("vk/constants.rs");
+    let mut vk_extensions_file = File::create(dir.join("extensions.rs")).expect("vk/extensions.rs");
+    let mut vk_feature_extensions_file =
+        File::create(dir.join("feature_extensions.rs")).expect("vk/feature_extensions.rs");
+    let mut vk_const_debugs_file =
+        File::create(dir.join("const_debugs.rs")).expect("vk/const_debugs.rs");
+    let mut vk_aliases_file = File::create(dir.join("aliases.rs")).expect("vk/aliases.rs");
+
+    let feature_code = quote! {
+        use std::os::raw::*;
+        use crate::vk::bitflags::*;
+        use crate::vk::definitions::*;
+        use crate::vk::enums::*;
         #(#feature_code)*
+    };
+
+    let definition_code = quote! {
+        use crate::vk::prelude::*;
+        use std::fmt;
+        use std::os::raw::*;
+        use crate::vk::aliases::*;
+        use crate::vk::bitflags::*;
+        use crate::vk::constants::*;
+        use crate::vk::enums::*;
         #(#definition_code)*
+    };
+
+    let enum_code = quote! {
+        use std::fmt;
         #(#enum_code)*
+    };
+
+    let bitflags_code = quote! {
+        use crate::vk::definitions::*;
         #(#bitflags_code)*
+    };
+
+    let constants_code = quote! {
+        use crate::vk::definitions::*;
         #(#constants_code)*
+    };
+
+    let extension_code = quote! {
+        use crate::vk::prelude::*;
+        use std::os::raw::*;
+        use crate::vk::aliases::*;
+        use crate::vk::bitflags::*;
+        use crate::vk::definitions::*;
+        use crate::vk::enums::*;
         #(#extension_code)*
-        #feature_extensions_code
+    };
+
+    let feature_extensions_code = quote! {
+        use crate::vk::bitflags::*;
+        use crate::vk::enums::*;
+       #feature_extensions_code
+    };
+
+    let const_debugs = quote! {
+        use std::fmt;
+        use crate::vk::bitflags::*;
+        use crate::vk::definitions::*;
+        use crate::vk::enums::*;
         #const_debugs
+    };
+
+    let aliases = quote! {
+        use crate::vk::bitflags::*;
+        use crate::vk::definitions::*;
+        use crate::vk::enums::*;
         #(#aliases)*
     };
-    write!(&mut file, "{}", source_code).expect("Unable to write to file");
+
+    let vk_rs_code1 = r#"
```

Comment:
Can you give those vars descriptive names like `clippy_lints` and  `exports` or similar instead of vr_rs_code12?